tldraw/tldraw
# Fix newlines in text geo shapes

Previously, entering a geo shape and adding a bunch of newlines wouldn't correctly update the shape's height. This fixes that.

### Change Type



### Test Plan

1. Create a geo shape
2. Enter text editing mode
3. Spam the enter key
4. See that the shape grows appropriately
5. Exit editing and check that the trailing newlines have been deleted.
